Sources/FoundationEssentials/Error/ErrorCodes.swift

```diff
@@ -30,7 +30,11 @@ extension CocoaError.Code {
         return CocoaError.Code(rawValue: 259)
     }
     public static var fileReadNoSuchFile: CocoaError.Code {
+#if os(Windows)
+        return CocoaError.Code(rawValue: 4)
```

Comment:
This feels like the wrong place to adjust this, this would cause `.fileNoSuchFile == .fileReadNoSuchFile` to be true on Windows but false on other platforms - I suspect this could lead to unexpected behavior in cross-platform clients.

Should we instead update the mapping from Windows error codes to `CocoaError.Code` in order to account for this, or should we update the location where the error is thrown to throw a different error?

Reply:
I don't think that mapping is the right answer - the error codes may be used in different ways in different APIs. Updating the location where the error is thrown might be reasonable though.